scripts/phil.sh

```diff
@@ -0,0 +1 @@
+./scripts/build_platform_images.sh -p test -d dev && ./scripts/deploy_platform.sh -p test -d dev && ./scripts/restart_platform_pods.sh -p test -d dev
```

Comment:
`phil.sh`?

If the deployment doesn't yet exist, this will zap the containers that are starting for the first time. ie. if the deploy_platform has any effect, it will restart the pods anyways...

Reply:
opps! i'll remove that now